lib/livebook_web/live/layout_helpers.ex

```diff
@@ -312,4 +312,31 @@ defmodule LivebookWeb.LayoutHelpers do
   defp topbar_class(:warning), do: "bg-yellow-200 text-gray-900"
   defp topbar_class(:info), do: "bg-blue-200 text-gray-900"
   defp topbar_class(:error), do: "bg-red-200 text-gray-900"
+
+  def hub_header(assigns) do
```

Comment:
Perhaps call it `teams_header`? I would also move this to `livebook_web/components/teams_components.ex`. I think the helpers nomenclature has been mostly removed from Phoenix and we should start aligning ourselves accordingly. :)

WDYT @jonatanklosko?

If agreed, we can rename the other `_helpers.ex`modules later.

Reply:
Definitely! There are a couple genuine helpers with no components, but otherwise moving sounds good :)